volpiano_display_utilities/volpiano_syllabification.py

```diff
@@ -111,7 +111,10 @@ def syllabify_volpiano(volpiano: str) -> Tuple[List[SyllabifiedVolpianoSection],
             vol_syls = VOLPIANO_SYLLABLE_REGEX.findall(vol_word)
             # Check if the last syllable of the word is improperly spaced
             # with three and only three trailing hyphens.
-            if vol_syls[-1][-3:] != "---" or vol_syls[-1][-4] == "-":
+            final_syl_of_word = vol_syls[-1]
+            if final_syl_of_word[-3:] != "---" or len(final_syl_of_word) < 4:
+                volpiano_improperly_spaced = True
+            elif vol_syls[-1][-4] == "-":
```

Comment:
I don't really understand the purpose of this change - maybe I'm thrown off by the commit message - if `len(vol_syls) < 4`, we would have gotten an `IndexError` rather than a `ValueError` when we checked `vol_syls[-1][-4]` wouldn't we have?

For the elif at the bottom here, could we not have `elif final_syl_of_word[-4] ...` instead of `elif vol_syls[-1][-4] ...`?

A simpler alternative, which I understand to be more pythonic, would be to wrap `if vol_syls[-1][-3:] != "---" or vol_syls[-1][-4] == "-":` in a `try: ... except IndexError: # last syllable has too few characters ...`

But I digress - if this is just a typo in the commit message, that's fine - don't try to fix the commit message. Switching to a `try/except` is up to you, whichever approach you prefer is fine.

Reply:
Ok, a few things happened here:

1. I just was careless in my commit message and wrote `ValueError` rather than `IndexError`...oops!
2. I switched to defining the `final_syl_of_word` variable rather than using `vol_syls[-1]` everywhere because it seemed clearer...and then forgot to replace every instance of `vol_syls[-1]`...also oops!
3. The `try`/`except` approach does seem like it makes more sense generally. I think I'll just make that switch here.

Reply:
so....standby for some fixes.